toolkit/indoors/src/main/java/com/arcgismaps/toolkit/indoors/FloorFilter.kt

```diff
@@ -127,25 +140,103 @@ public fun FloorFilter(
+                }
+            }
+        }
+    }
+}
 
-            // displays only the selected floor when enabled
-            var isFloorsCollapsed by rememberSaveable { mutableStateOf(false) }
+@Composable
+internal fun FloorFilterContent(floorFilterState: FloorFilterState, uiProperties: UIProperties, modifier: Modifier) {
+    Column(
+        modifier = modifier,
+        verticalArrangement = Arrangement.Center
+    ) {
+        // displays only the selected floor when enabled
+        var isFloorsCollapsed by rememberSaveable { mutableStateOf(false) }
 
-            // boolean toggle to display the site facility selector dialog
-            var isSiteAndFacilitySelectorVisible by rememberSaveable { mutableStateOf(false) }
+        // boolean toggle to display the site facility selector dialog
+        var isSiteAndFacilitySelectorVisible by rememberSaveable { mutableStateOf(false) }
 
-            // boolean toggle to display either the sites selector or the facilities selector in the site facility selector dialog,
-            // display sites selector by default when set to false, and sites selector when set to true.
-            var isFacilitiesSelectorVisible by rememberSaveable { mutableStateOf(false) }
+        // boolean toggle to display either the sites selector or the facilities selector in the site facility selector dialog,
+        // display sites selector by default when set to false, and sites selector when set to true.
+        var isFacilitiesSelectorVisible by rememberSaveable { mutableStateOf(false) }
 
-            // get the current selected facility
-            val selectedFacility = floorFilterState.onFacilityChanged.collectAsStateWithLifecycle().value
+        // get the current selected facility
+        val selectedFacility =
+            floorFilterState.onFacilityChanged.collectAsStateWithLifecycle().value
 
-            // get the selected level ID
-            val selectedLevelID = floorFilterState.onLevelChanged.collectAsStateWithLifecycle().value?.id
+        // get the selected level ID
+        val selectedLevelID =
+            floorFilterState.onLevelChanged.collectAsStateWithLifecycle().value?.id
 
-            // if no facility is selected, only display site-facility selector button
-            if (selectedFacility == null) {
+        // if no facility is selected, only display site-facility selector button
+        if (selectedFacility == null) {
+            SiteFacilityButton(
+                modifier,
+                floorFilterState,
+                isSiteAndFacilitySelectorVisible,
+                isFacilitiesSelectorVisible,
+                uiProperties,
+                onSiteFacilitySelectorVisibilityChanged = { isVisible ->
+                    isSiteAndFacilitySelectorVisible = isVisible
+                },
+                onFacilitiesSelectorVisible = { isVisible ->
+                    isFacilitiesSelectorVisible = isVisible
+                }
+            )
+            return
```

Comment:
This was previously `return@Surface`
```suggestion
            return@Column
```